documentation/src/main/asciidoc/upgrading/upgrading.asciidoc

```diff
@@ -7,6 +7,17 @@ The Infinispan community
 
 This guide walks you through the process of upgrading Infinispan.
 
+== Upgrading from 9.3 to 9.4
+
+=== Store Changes
+
+==== Shared and Passivation
+
+A store cannot be configured as both shared and having passivation enabled.
```

Comment:
Was just wondering, don't we also have a doc section for passivation where such fundamental limitations/implications of passivation should also be mentioned, besides the 'Upgrading' section?

Reply:
👍 let me make sure that is added.